aws-servicecatalog-serviceaction/aws-servicecatalog-serviceaction.json

```diff
@@ -94,6 +94,11 @@
       "permissions": [
         "servicecatalog:DeleteServiceAction"
       ]
+    },
+    "list": {
+      "permissions": [
+        "servicecatalog:ListServiceActionsPaginator"
```

Comment:
Is this right? Do we have to specifify the paginator? What about the FAS policy?

Reply:
Updated permission